contracts/dao-core/schema/dao-core.json

```diff
@@ -1023,7 +1023,7 @@
                     ]
                   },
                   "channel_id": {
-                    "description": "exisiting channel to send the tokens over",
+                    "description": "existing channel to send the tokens over",
```

Comment:
this is, unfortunately, outside the domain of DAO DAO. you can make a PR to CosmWasm though if you would like to fix it!

https://github.com/CosmWasm/cosmwasm/blob/2eb97078418ca81a8dd1858871681ab97c4cd079/packages/std/src/ibc.rs#L30